packages/react-instantsearch-hooks-dom/src/ui/Breadcrumb.tsx

```diff
@@ -47,12 +47,17 @@ type UseBreadcrumbRenderState = ReturnType<typeof useBreadcrumb>;
 
 type BreadcrumbItem = UseBreadcrumbRenderState['items'][number];
 
-export type BreadcrumbProps = React.HTMLAttributes<HTMLDivElement> &
+type WrapperProps = React.ComponentProps<'div'>;
+
+export type BreadcrumbWidgetProps = WrapperProps & {
```

Comment:
The UI components shouldn't be aware of widgets, even conceptually.

Reply:
Could that be solved with a change of naming? Having types in a single place addresses an earlier feedback, making it less difficult to have correct typing for a component.

With a less specific name like `BreadcrumbForwardedProps` or something like that, we remove the vocabulary specific to widget, while retaining type consistency for the component. Wdyt?